runtime-testsuite/test/org/antlr/v4/test/runtime/ExtraTests.java

```diff
@@ -0,0 +1,68 @@
+package org.antlr.v4.test.runtime;
+
+public class ExtraTests {
```

Comment:
Sorry to be fussy but "Extra" isn't very specific. Maybe "GeneratedTests" or even "GeneratedLexerDescriptors"?

Reply:
I'll merge then think/tweak of name

Reply:
Yes, but I haven't found a better name. Generated is also not very correct since it contains line separator tests. Maybe handwritten or something like this.